go/host/storage/interfaces.go

```diff
@@ -44,6 +44,8 @@ type BatchResolver interface {
 	FetchBatchByHeight(height *big.Int) (*common.PublicBatch, error)
 	// FetchTotalTxCount returns the number of transactions in the DB
 	FetchTotalTxCount() (*big.Int, error)
+	// FetchTotalTxsQuery returns the number of transactions in the DB
```

Comment:
Need to give this a different comment to the one above imo, clarify why these two identical looking methods exist a bit haha.